app/src/androidTest/java/ch/sdp/vibester/activity/LyricsBelongGameActivityTest.kt

```diff
@@ -1,30 +1,35 @@
 package ch.sdp.vibester.activity
 
+import android.content.Context
 import android.content.Intent
 import androidx.test.core.app.ActivityScenario
 import androidx.test.core.app.ApplicationProvider
 import androidx.test.espresso.Espresso.onView
-import androidx.test.espresso.action.ViewActions.click
 import androidx.test.espresso.assertion.ViewAssertions.matches
 import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
 import androidx.test.espresso.matcher.ViewMatchers.*
 import androidx.test.ext.junit.rules.ActivityScenarioRule
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import androidx.test.rule.GrantPermissionRule
 import ch.sdp.vibester.R
-import org.hamcrest.CoreMatchers.containsString
-import org.junit.After
-import org.junit.Before
-import org.junit.Rule
-import org.junit.Test
+import ch.sdp.vibester.api.ItunesMusicApi
+import ch.sdp.vibester.api.LastfmMethod
+import ch.sdp.vibester.helper.GameManager
+import ch.sdp.vibester.model.Song
+import okhttp3.OkHttpClient
+import org.hamcrest.CoreMatchers.not
+import org.junit.*
```

Comment:
I believe it's a good practice to know exactly what are you importing. So maybe change * to the functions you actually are importing

Reply:
I think this is some optimization done automatically by Android Studios when I am using too many functions from the same class to "save space"

Reply:
Okey then